testing/ettercap/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Karim Kanso <kaz.kanso@gmail.com>
+# Maintainer: Karim Kanso <kaz.kanso@gmail.com>
+pkgname=ettercap
+pkgver=0.8.2_git20190428
+_gitrev=5fb604823c6063676ebdf7cac66b7807acb4f7ed
+pkgrel=0
+pkgdesc="Multipurpose sniffer/interceptor/logger for switched LAN"
+url="https://www.ettercap-project.org/"
+arch="all"
+license="GPL-2.0-or-later"
+depends=""
+makedepends="cmake make gcc musl-dev ncurses-dev curl-dev libpcap-dev
+	     libnet-dev pcre-dev flex bison linux-headers geoip-dev
+	     check-dev"
+install=""
+subpackages="$pkgname-doc"
+source="ettercap-$pkgver.zip::https://github.com/Ettercap/ettercap/archive/$_gitrev.zip"
+builddir="$srcdir"/$pkgname-$_gitrev
+
+build() {
+	cd "$builddir"
```

Comment:
the cd "$builddir" statements can be removed as it is done implicitly now by `abuild`

Reply:
Thanks, didn't realise this.